### PR TITLE
Up and down arrow work for notebook bar

### LINKED ISSUE #6933: Up and down arrow keys should move focus up/down a level
## Description
Not possible to use Up/Down arrow to navigate in between tab and tab's content.

### Mentors:
@Minion3665 

## To reproduce
1. `make run`
2. Switch to tabbed view if not already ( on the compact view: view > tabbed view)
3. Press `Alt` (chrome based browsers) or `Shift + Alt` (FF, etc)
4. AccessKey hints will appear
5. Press 'N' to switch to 'Insert' tab
6. Press `Down` arrow key
7. Focus outline appears surrounding one of that tab's buttons
8. Now press 'Up' arrow key
9. Nothing happens

It should move the focus back to the upper level: Active Tab.

Example: a pressed down arrow key when the focus is in the "Home" tab moves the focus to the first item inside of that tab

## Code pointer
Part of the solution may reside between these lines:
* https://github.com/CollaboraOnline/online/blob/1d7be47f0356b4a4d2b302c28c531551237bc86a/browser/src/control/Control.JSDialogBuilder.js#L1277

For going down at least. For going up, some more search may be needed :) according to @gokaysatir 

Also the following PR https://github.com/CollaboraOnline/online/pull/6823/files doesn't really work. You can test it and see that the focus never gets back to the tab.

### PR DESCRIPTION
**Pull request by CosmicOli**
Resolves #6933

Change-Id: I1cf696223ff5739e6a48095acbdf3fd778af1ffd

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Review by Darshan-upadhyay1110 — CHANGES_REQUESTED** [review]
- When document get loaded first time `down arrow key` is not working 
-  and also can we do one thing here like keep function seperated like no decalration of function inside other function. this is verymuch helpful if we need that child function sepratly to use in future ( maybe :) );

**Comment by Darshan-upadhyay1110** [comment]
@CosmicOli thanks for the fix. Please check my comment above and let me know for any confusion here.

File: browser/src/dom/NotebookbarAccessibility.js [hunk 1]
```diff
@@ -356,6 +356,10 @@ var NotebookbarAccessibility = function() {
 					  this.focusToMap();
 					}
 				  }.bind(this));
+
+				if (element.classList.contains('selected')) {
+					this.setupAcceleratorsForCurrentTab(element.id);
+				}
```
Comment (gokaysatir) [inline browser/src/dom/NotebookbarAccessibility.js:362]: Thanks for the improvements :)
The reason we removed this from here was: "to not assign accelerator keys before a tab is selected."
If we add this again, there may be conflicts with the tab header accelerator keys.

I think we can modify below part:
    "else if (key === 'ARROWDOWN') {"

We can make it work regardless of the active tab pointers content list. You can check the current tab and find its content and focus on the first element. Without using "this.activeTabPointers.contentList" or any other variable from within the class. Can you check if it is possible?

I will be continuing with the second commit.

**Review by gokaysatir — CHANGES_REQUESTED** [review]
For the second commit:
I don't think we need this many lines of code. It looks to me like we could add a handler to "toolbar-wrapper" element. That element could catch the "ArrowUp" event and focus on the tab header. It would take only a few lines.

Can you check if it is possible?

**Review by gokaysatir — APPROVED** [review]
Cool :) Thanks for the improvements.

One note as a one-line follow up. When the focus is inside "accessibilityInputElement", autocomplete feature of the browser is activated. Using down and up arrows triggers an autocompletion list.

To prevent this, we can add below line inside initAccessibilityInputElement function:
* this.accessibilityInputElement.autocomplete = 'off';

Thanks :)

**Comment by Darshan-upadhyay1110** [comment]
thanks @CosmicOli for the fix.
there is merge confilicts can you please resolve and update the branch with master ?

**Review by Darshan-upadhyay1110 — APPROVED** [review]

**Comment by pedropintosilva** [comment]
@CosmicOli :
* There is a "merge branch" commit that shouldn't be here  
* Also it seems you didn't rebase your changes on top of latest master

```
*   dcf5e8bfb1	CosmicOli	Fri Aug 18 10:13:56 2023 +0100	Merge branch 'master' into private/oli/up-and-down-arrow
|\  
| * c3aaae077f	Darshan-upadhyay1110	Wed Aug 16 14:33:26 2023 +0530	Add missing dark mode toggle in mobile view
| * 3103ab5c66	Pedro Pinto Silva	Thu Aug 17 13:34:23 2023 +0200	Add missing rename document icon
| * a1a6817e4d	Aron Budea	Thu Aug 17 02:56:54 2023 +0200	unocommands.py: Treat JS/TS files as UTF-8
| * 89f283c676	Caolán McNamara	Wed Aug 16 21:33:53 2023 +0100	cool#7102 fix 2nd presentation under chrome not appearing
| * 851c232531	Caolán McNamara	Wed Aug 16 11:16:32 2023 +0100	cid#318843 COPY_INSTEAD_OF_MOVE
| * 62504735ea	Caolán McNamara	Tue Aug 15 16:13:58 2023 +0100	cid#318868 silence bogus Double free
| * a45b4e875f	Gökay Şatır	Wed Aug 16 13:09:44 2023 +0300	Add accessibility keys for insert and format tabs.
| * 10b58a0a4f	Gökay Şatır	Wed Aug 16 12:07:59 2023 +0300	Add accessibility definitions to draw UI.
| * 9fd60f55e0	Gökay Şatır	Wed Aug 16 12:47:36 2023 +0300	Update Cypress tests.
| * b304644be4	Gökay Şatır	Tue Aug 15 15:25:33 2023 +0300	Calc - Add remaining tabs' accessibility definitions.
| * 9436512afb	Gökay Şatır	Tue Aug 15 13:53:03 2023 +0300	Calc File and Home tabs' accessibility definitions.
| * 33abd9c437	Gökay Şatır	Tue Aug 15 13:52:07 2023 +0300	Initialize contentlist with empty array.
| * 49354607e0	Gökay Şatır	Tue Aug 15 12:35:53 2023 +0300	Re-enable multi-language support on accessibility definitions.
| * 34635de5d4	Gökay Şatır	Tue Aug 15 12:11:35 2023 +0300	Calc: Add ids to notebookbar definitons & simplify definitons.
| * ac8440fea7	Szymon Kłos	Tue Aug 15 14:17:15 2023 +0200	jsdialog: add minimal size for treeview
| * c4183d2709	Rash419	Mon Aug 14 21:43:17 2023 +0530	android: fix app crashes after closing the document
| * 6a7b98af8a	Rash419	Tue Aug 15 21:26:18 2023 +0530	fix: cell border preview in mobile wizard
| * 639068e3fe	Rash419	Tue Aug 15 18:42:23 2023 +0530	impress: remove unused mobile-slide-sorter - to show previews we use slide-sorter element   and presentation-controls-wrapper as container
| * 03fa9d9c86	Szymon Kłos	Tue Aug 15 12:19:29 2023 +0200	jsdialog: always execute close message from server
| * 2eda4740ca	Szymon Kłos	Tue Aug 15 09:32:33 2023 +0200	Don't show presentation on iOS in small frame
| * 9a0b34f8fb	Szymon Kłos	Tue Aug 15 14:34:40 2023 +0200	jsdialog: fix error on insert_button
| * 42df0dac71	Caolán McNamara	Tue Aug 15 11:31:45 2023 +0100	cid#318842 silence Unchecked return value
| * ce0c11f09f	Caolán McNamara	Tue Aug 15 11:23:23 2023 +0100	cid#318866 can elide the temporary here
| * 461fe6a630	Caolán McNamara	Tue Aug 15 11:19:17 2023 +0100	cid#318940 Big parameter passed by value
| * eaa2b0cf2f	Caolán McNamara	Mon Aug 14 14:59:58 2023 +0100	cid#318821 COPY_INSTEAD_OF_MOVE
| * 5dcf65f5c6	Caolán McNamara	Mon Aug 14 14:58:37 2023 +0100	cid#318893 COPY_INSTEAD_OF_MOVE
| * 68135a4682	NickWingate	Mon Aug 14 14:44:55 2023 +0100	Rename with file default when given unspecified extension
| * 24ac74db4f	NickWingate	Mon Jul 31 16:41:57 2023 +0100	Add rename document to notebookbar and menubar
| * 478fd19ccc	Caolán McNamara	Mon Aug 14 12:48:02 2023 +0100	cid#318850 Unchecked return value
| * 3502e1d006	Caolán McNamara	Mon Aug 14 12:21:36 2023 +0100	cid#318903 Dereference after null check
| * 6062342827	Caolán McNamara	Mon Aug 14 12:15:26 2023 +0100	cid#318980 COPY_INSTEAD_OF_MOVE
| * 7d6619be90	Miklos Vajna	Mon Aug 14 08:19:13 2023 +0200	wsd, coding style: document double review for security
| * 41bdf1ece8	Rash419	Mon Aug 14 21:33:55 2023 +0530	admin: remove fixed width and height attr from svg
| * 65f6f46093	Rash419	Mon Aug 14 19:34:14 2023 +0530	admin: use humanizeMem in adminClusterOverview.js
| * 555a07db70	Pedro Pinto Silva	Mon Aug 14 15:58:45 2023 +0200	AdminCluster: Use proper card headers for title instead
| * efe874f89c	Ashod Nakashian	Sun Aug 13 08:35:16 2023 -0400	wsd: simplify shutdown and termination flagging
| * 35dda35f9d	Ashod Nakashian	Sat Aug 12 16:57:04 2023 -0400	wsd: move statics into anonymous namespace
| * e660bb8e66	Gökay Şatır	Mon Aug 14 14:21:05 2023 +0300	When document layer is not ready and socket message is not delayed, message is lost.
| * 8570a03555	Gökay Şatır	Mon Aug 14 12:27:13 2023 +0300	Correct menubutton control id calculation.
| * 3d1725ea44	Gökay Şatır	Fri Aug 11 17:47:29 2023 +0300	Cypress updates.
| * 7d084e40cb	Gökay Şatır	Thu Aug 10 15:45:29 2023 +0300	Update cypress tests.
| * a3d9abcc20	Gökay Şatır	Thu Aug 10 10:37:54 2023 +0300	Move all accessibility definitions from notebookbaraccessibilitydefinitions into control.notebookbarwriter.
| * 7d0f7d7378	Gökay Şatır	Wed Aug 9 11:21:25 2023 +0300	writer insert tab keys are added. Replaced id based checks with uno command ones. Removed unnecessary function from notebookbaraccessibility. We don't use automatic id assignments any more.
| * 1a2500c8d3	Gökay Şatır	Tue Aug 8 15:46:49 2023 +0300	Merge accessibility definitions with notebookbar definitions.
| * 939a7a0301	Gökay Şatır	Mon Aug 7 16:38:46 2023 +0300	Simplify notebookbarwriter and tab page containers.
| * 2cc714705a	Gökay Şatır	Mon Aug 7 16:05:26 2023 +0300	Remove access key assignments from jsdialogs.
| * f8e53bd851	Gökay Şatır	Mon Aug 7 15:31:21 2023 +0300	Assign tab ids while defining tabs.
| * 6682a2a527	Caolán McNamara	Thu Aug 10 09:56:56 2023 +0100	cid#318900 Out-of-bounds write
| * 307818ba1c	Caolán McNamara	Thu Aug 10 09:42:00 2023 +0100	cid#318834 Missing move assignment operator
| * f7e121e2ea	Caolán McNamara	Thu Aug 10 09:37:41 2023 +0100	cid#318953 COPY_INSTEAD_OF_MOVE
| * 4b9cbab33d	Szymon Kłos	Fri Aug 11 09:17:01 2023 +0200	jsdialog: fix spinfield precision detection
| * eee6e49e8a	Szymon Kłos	Sat Aug 12 12:52:39 2023 +0200	cypress: fix unstable alignment test
| * 697f4a6109	Szymon Kłos	Sat Aug 12 10:10:06 2023 +0200	cypress: fix failing table tests
| * b85eb1b03f	Szymon Kłos	Fri Aug 11 20:55:44 2023 +0200	cypress: fix svg text-decoration detection
| * e6a06eae1c	Shehr Bano	Wed Aug 9 22:58:50 2023 +0500	FIX #7037: JSDialog: label vs textarea vs input field inconsistent paddings
| * 16b8bead39	Pedro Pinto Silva	Fri Aug 11 11:35:07 2023 +0200	Revert "Add action for detecting code duplicates"
| * 47f4305050	Rash419	Thu Aug 10 13:39:45 2023 +0530	fix: cursor jump when hyperlink popup clicked in mobileview
| * 912dfcd08e	Caolán McNamara	Wed Aug 9 21:49:20 2023 +0100	autofill onMouseDown doesn't generate working event position for hidpi
| * a42adac98f	Ashod Nakashian	Fri Aug 4 05:30:57 2023 -0400	wsd: add the web-server URL to the media-src list
| * 85ee03f98f	Andras Timar	Wed Aug 9 20:01:33 2023 +0200	CollaboraOnline#6546 enable automatic color in default paragraph style
| * c89cccfc7a	Pedro Pinto Silva	Fri Aug 4 12:58:49 2023 +0200	Welcome: allow scroll bar
| * 96a7cfb25e	Marco Cecchetti	Mon Jul 10 12:14:17 2023 +0200	a11y: provide info about current table and cell to screen reader
| * 7825f82ce1	Darshan-upadhyay1110	Wed Aug 9 18:10:32 2023 +0530	Remove open_local_editor button in tablet view
| * 5185b7bedf	Caolán McNamara	Wed Aug 9 14:18:47 2023 +0100	cid#318934 silence Untrusted loop bound
| * d79d86ed91	Caolán McNamara	Wed Aug 9 14:10:21 2023 +0100	cid#318844 Dereference before null check
| * 2a8910a941	Caolán McNamara	Wed Aug 9 14:05:33 2023 +0100	addPollFor default arg is now always explicitly passed
| * c593a0bde5	Caolán McNamara	Wed Aug 9 14:04:57 2023 +0100	replaySync is unused
| * b6f87b06e3	Caolán McNamara	Wed Aug 9 13:59:41 2023 +0100	cid#322475 COPY_INSTEAD_OF_MOVE
| * 1cd88f217e	Caolán McNamara	Wed Aug 9 13:52:02 2023 +0100	cid#321976 AUTO_CAUSES_COPY
| * 5dcce1965e	Rash419	Wed Aug 9 14:59:08 2023 +0530	android: enable dark mode in documents
| * c2192b4a67	Pedro Pinto Silva	Wed Aug 9 13:13:05 2023 +0200	Screen reader doesn't properly report tab names in Tab View UI
| * 0c611649e5	Caolán McNamara	Fri Aug 4 16:28:07 2023 +0100	cid#318854 Unchecked return value
| * 6d873d6837	Caolán McNamara	Fri Aug 4 10:13:30 2023 +0100	cid#318919 Dereference after null check
| * b1f88c4cb2	Caolán McNamara	Fri Aug 4 10:08:00 2023 +0100	cid#318875 Dereference after null check
| * 2119a19881	Caolán McNamara	Fri Aug 4 09:57:30 2023 +0100	cid#321981 AUTO_CAUSES_COPY
| * a2b924033d	Henry Castro	Mon Aug 7 09:19:09 2023 -0400	admin: rename ACCESS_TOKEN -> JWT_TOKEN
| * 441d064b7a	Gökay Şatır	Fri Aug 4 14:08:25 2023 +0300	Make insert shapes menu accessible.
| * 340bbb0328	Rash419	Mon Aug 7 15:00:28 2023 +0530	admin: increase font-size of y-axis
| * 3f0c9d5c55	Pedro Pinto Silva	Mon Aug 7 09:55:54 2023 +0200	admin: imprv ui sizes of adminClusterOverview.html
| * c12885c6ec	Gökay Şatır	Thu Aug 3 15:07:13 2023 +0300	Make insert table popup accessible.
| * 345c8c7d48	Gökay Şatır	Thu Aug 3 14:42:04 2023 +0300	Fix js error - notebookbar accessibility.
| * a003126a65	Paris Oplopoios	Thu Aug 3 19:00:19 2023 +0300	Give new sockets a canonical view id in tests
| * d0c5321aaf	Paris Oplopoios	Sat Jul 22 22:21:47 2023 +0300	Fix getting canonical view id of 1000
| * 0ed983d7a3	Rash419	Wed Aug 2 12:25:34 2023 +0530	admin: refactor adminClusterOverview.js
| * 87bc8b10c4	Rash419	Tue Aug 1 20:57:30 2023 +0530	admin: imprv ui of adminClusterOverview.html
| * 8a879404ba	Rash419	Tue Aug 1 19:07:16 2023 +0530	admin: added podname to card
| * 944f269926	Rash419	Tue Aug 1 17:28:09 2023 +0530	admin: added new cluster overview page
| * fc77e1a1ff	Rash419	Fri Aug 4 18:30:34 2023 +0530	admin: fix auth issue when switching between        multiple cool servers - also good to check response first as client   might use persisted cookies
| * 718992e30a	Caolán McNamara	Wed Aug 2 09:14:45 2023 +0100	cool#6911 don't show notes in hidden cells
| * 15422721b7	Caolán McNamara	Tue Aug 1 21:22:12 2023 +0100	cool#6911 don't show off-screen notes from col/row 0
| * 7b51f8c22e	Caolán McNamara	Tue Aug 1 15:29:09 2023 +0100	cool#6911 draw the note indicator browser-side
| * 4d7f7d67d0	Caolán McNamara	Mon Jul 31 11:31:12 2023 +0100	cool#6911 calc comment positions are now cell addresses
| * aaa61e44e5	Caolán McNamara	Thu Aug 3 10:59:15 2023 +0100	allow tileprocessed to accept multiple comma-seperated tileids
| * 509c29421e	Caolán McNamara	Thu Aug 3 09:30:26 2023 +0100	split this hunk into reusable ClientSession::onTileProcessed
| * 040ee51ec9	Caolán McNamara	Fri Aug 4 10:57:28 2023 +0100	occasionally testTileSubscription fails
| * 5485bea24e	Gökay Şatır	Fri Aug 4 09:56:56 2023 +0300	Add arrow key handlers for better accessibility support.
| * 14c84ac453	Gökay Şatır	Wed Aug 2 15:48:23 2023 +0300	Improve accessibility of w2ui drop down menus.
| * 8bb41ee775	Caolán McNamara	Thu Aug 3 21:19:43 2023 +0100	cid#318947 Unchecked return value from library
| * 9e0b0a0c85	Caolán McNamara	Thu Aug 3 21:01:32 2023 +0100	cid#318909 Argument cannot be negative
| * 1d179cdf76	Caolán McNamara	Thu Aug 3 20:59:41 2023 +0100	cid#321981 AUTO_CAUSES_COPY
| * 7cb1dfd4eb	Caolán McNamara	Thu Aug 3 20:58:48 2023 +0100	cid#321977 AUTO_CAUSES_COPY
| * c56ebcc06d	Henry Castro	Wed Aug 2 17:01:56 2023 -0400	browser: remove hyperlink pop-up
| * cbeee7727e	Pranam Lashkari	Thu Jul 27 00:00:13 2023 +0700	accessibilty: added accessibilty fields to the mobile edit button
| * e8b2ea5ca1	Szymon Kłos	Fri Jul 28 13:49:06 2023 +0200	Deduplicate onInitialize method in Column/Row Groups
| * 8ce751a3ca	Szymon Kłos	Mon Jul 3 11:33:04 2023 +0200	Add action for detecting code duplicates
| * 3831e6f4e2	Rash419	Thu Aug 3 16:05:03 2023 +0530	docker: fix permission denied in archlinux
| * b38a91d1c5	Darshan-upadhyay1110	Thu Aug 3 12:44:38 2023 +0530	Add check for Insert_buttons to display in tablet view
| * 1428888afc	Shehr Bano	Thu Aug 3 16:39:19 2023 +0500	Writer: Fix to issue #6977, extends vertical space of ui-treeview in Manage Changes
| * c9a9e5635d	Rash419	Wed Jul 26 13:44:03 2023 +0530	show copy/paste timeout warning
| * 63473360b1	Jaume Pujantell	Fri Jul 14 09:48:13 2023 +0200	save-as preserves document password
| * 75ae9894b9	Ashod Nakashian	Tue Aug 1 07:21:20 2023 -0400	wsd: Util::make_unique -> std::make_unique
| * 347b3c90e5	Ashod Nakashian	Sun Jul 30 18:46:11 2023 -0400	wsd: correct assertion
| * 5b52f7d8eb	Ashod Nakashian	Sun Jul 30 10:40:21 2023 -0400	wsd: smart pointer cleanup
| * 67dd4e4a9d	Ashod Nakashian	Sat Jul 29 13:10:31 2023 -0400	wsd: test: new cool.html post request test
| * 35605510aa	Ashod Nakashian	Sat Jul 29 10:31:02 2023 -0400	wsd: include string_view
| * b8854ea2e9	Ashod Nakashian	Tue Jun 20 06:35:43 2023 -0400	wsd: decouple FileServerRequestHandler from COOLWSD
| * d47001267b	Ashod Nakashian	Tue Jun 20 05:53:48 2023 -0400	wsd: reduce manual http header creation
| * 5e8707bce9	Ashod Nakashian	Tue May 16 17:47:40 2023 -0400	make: better clang detection
| * 4f8d0fc4cb	Ashod Nakashian	Wed May 31 06:42:40 2023 -0400	wsd: test: better UnitWSDClient interface
| * c1725faac1	Ashod Nakashian	Mon Jul 17 07:29:22 2023 -0400	wsd: dump the last quarantined version with the status
| * fbc4373ceb	Ashod Nakashian	Wed Jul 12 07:12:15 2023 -0400	gcov: support cleaning coverage data and document
| * b9a1254126	Henry Castro	Mon Jul 31 12:15:24 2023 -0400	browser: admin: use access token
| * 47485adc3e	Henry Castro	Mon Jul 31 12:10:20 2023 -0400	admin: add access token
| * 1f5919140d	Maks Wolkowinski	Fri Jul 28 14:57:55 2023 +0200	Collapse notebook bar from view tab
| * d99aece12f	Marco Cecchetti	Tue Aug 1 11:42:26 2023 +0200	Improve a11y support for tabbed UI
| * 7afe5dfe8b	Gökay Şatır	Wed Aug 2 12:54:15 2023 +0300	Remove tabIndex from container of color pickers. Turn image buttons into actual buttons.
| * e414ac3c86	Gökay Şatır	Wed Aug 2 11:57:45 2023 +0300	Rename conflicting accelerator keys.
| * 54b16ea222	Gökay Şatır	Tue Aug 1 09:57:52 2023 +0300	Make color buttons interactable with keyboard.
| * fd7e041c90	Gökay Şatır	Mon Jul 31 19:25:23 2023 +0300	Add focus loop to color control when keyboard is used.
| * a73ae5a1a5	Gökay Şatır	Fri Jul 7 13:24:32 2023 +0300	Focus on the color input popup when keyboard is used.
| * 3a8ff4ef3d	Maks Wolkowinski	Tue Aug 1 15:45:35 2023 +0200	feat: Add hide menu entry in compact mode
| * 325ef1be01	Gökay Şatır	Mon Jul 31 17:44:14 2023 +0300	Add functionality for setting access keys of buttons which are added via insertbutton post message.
| * 429f895bbf	Gökay Şatır	Mon Jul 31 17:22:20 2023 +0300	Allow notebookbar insertbutton post message to specify accessKey.
| * 76fc8f176a	Darshan-upadhyay1110	Fri Jul 28 14:30:40 2023 +0530	Add reset accessibilityKey
| * 498a02c522	Pedro Pinto Silva	Fri Jul 28 15:46:17 2023 +0200	Show userlist on tablets
| * 218583ce22	Pedro Pinto Silva	Mon Jul 31 10:35:58 2023 +0200	Tablet: Hide document file name input field
| * cf545e8152	Szymon Kłos	Mon Jul 31 15:06:47 2023 +0200	Don't steal focus from outside iframe on load Calc
| * 6c382f5415	Michael Meeks	Tue Jul 18 23:54:35 2023 +0100	rle: optimize inner loop bit-bashing.
| * 8a715cb04c	Caolán McNamara	Fri Jul 28 10:07:53 2023 +0100	calc insert comment checks undefined tab field
| * 03d83ab6b8	Michael Meeks	Fri Jul 14 16:59:25 2023 +0100	paste: cleanup excessive debugging.
| * ec67c92264	Pedro Pinto Silva	Thu Jul 27 10:14:15 2023 +0200	Tablet: Non-functional sidebar button shown in read-only mode
| * a56f034dc6	Szymon Kłos	Mon Jul 10 11:53:27 2023 +0200	Share code for findTailsGroup
| * 8b5df0c4b3	codewithvk	Sun Jul 16 06:21:01 2023 +0000	Fix : Menubutton should have disabled state styling different
| * 06f8ee610f	Tex2002ans	Wed Jul 26 17:06:11 2023 -0400	Update feature_request.md to use Headings
| * 6b79e52831	Tex2002ans	Wed Jul 26 17:04:47 2023 -0400	Update bug_report_smoke-test.md to use Headings
| * dcbc0208bb	Tex2002ans	Thu Jul 20 07:24:47 2023 -0400	Update bug_report.md
| * 850089377f	genofire	Tue Jul 25 14:47:13 2023 +0200	fix(helm): update appVersion
| * 16bb328456	Darshan-upadhyay1110	Tue Jul 25 17:52:42 2023 +0530	Added visual feedback after copy information into clipboard
* | 0c6b43d105	CosmicOli	Fri Aug 18 09:58:50 2023 +0100	Prevention of triggering the autocompletion list
* | 90815f5af8	CosmicOli	Mon Aug 14 10:20:21 2023 +0100	Moving the up arrow key event to tab pages
* | 9d6e458f62	CosmicOli	Thu Aug 10 10:41:14 2023 +0100	Removing use of tab pointers before tab selection
* | b355cd8ebe	CosmicOli	Fri Aug 4 11:40:58 2023 +0100	Fix accessibility up and down keys on first load
* | d5249251db	CosmicOli	Tue Jul 18 16:02:35 2023 +0100	Make up and down arrows move focus in notebook bar
|/  
* 2ac7183e44	Darshan-upadhyay1110	Wed Jul 19 14:13:14 2023 +0530	Add Left or right arrow on a tab label should switch tabs
```

**Pull request merged**